src/CoreBundle/Resources/config/commands.xml

```diff
@@ -2,9 +2,11 @@
 <container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
     <services>
         <service id="Sonata\CoreBundle\Command\SonataDumpDoctrineMetaCommand" class="Sonata\CoreBundle\Command\SonataDumpDoctrineMetaCommand">
+            <deprecated>The "%service_id%" service is deprecated since 3.x and will be removed in 4.0.</deprecated>
```

Comment:
```suggestion
            <deprecated>The "%service_id%" service is deprecated since sonata-project/core-bundle 3.x and will be removed in 4.0.</deprecated>
```
The same suggestion applies for the rest of deprecations.

Reply:
I will do it in Monday.